ovs/lib/setup.py

```diff
@@ -124,7 +124,7 @@ def setup_node(ip=None, force_type=None):
                 known_passwords = json.loads(config.get('setup', 'passwords'))
             if config.has_option('setup', 'external_etcd'):
                 external_etcd = json.loads(config.get('setup', 'external_etcd'))
-            enable_heartbeats = False
+            enable_heartbeats = config.getboolean('setup', 'enable_heartbeats')
```

Comment:
I would prefer to have the default set to true. See discussion of the BAM in OVS-3948